Agents/ESPHomeAgent/README.md

```diff
@@ -136,23 +136,49 @@ http://<IP address>:<port number>/<label>
 ```
 More information can be found in the `Dockerfile` in the same directory as this README section.
 
-To build and start the agent, open up the command prompt in the same directory as this README, run
+#### Agent deployment out of the stack
+
+To build and start the agent out of the stack, open up the command prompt in the same directory as this README, run
 ```
 docker-compose up -d
 ```
 
-The agent is reachable at "esphome-agent/toggle" on localhost port 1011.
+#### Stack deployment
+
+If you want to spin up this agent as part of a stack, instead of `docker-compose up -d`, do the following:
+- Copy the contents of `config/client.properties_stack` into `config/client.properties` and copy the contents of `config/esphome-client.properties_stack` into `config/esphome-client.properties`, inserting the name of your stack.
+- Build the image via `docker-compose build`. Do not start the container.
+- Copy the `json` file from the `stack-manager-input-config` folder into `TheWorldAvatar/Deploy/dynamic/stack-manager/inputs/config/services/`.
+- Go to the stack manager folder by following this route: `TheWorldAvatar/Deploy/dynamic/stack-manager/`, check whether there is a `<STACK NAME>.json` under the sub folder `/inputs/config/` and create one if it doesn't exist. If it exists already, append the agent to the json file. (Read [Stack configuration](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-manager#stack-configuration) for more information.)
```

Comment:
Deploy/stacks/dynamic

Reply:
Oh I made a mistake there. Thanks for pointing it out. I have edited the readme for both agents accordingly.